src/Route/DefaultRouteGenerator.php

```diff
@@ -116,7 +120,7 @@ public function generateMenuUrl(
         return [
             'route' => $this->caches[$code],
             'routeParameters' => $parameters,
-            'routeAbsolute' => $absolute,
+            'routeAbsolute' => UrlGeneratorInterface::ABSOLUTE_URL === $absolute,
```

Comment:
Isn't this a BC-break?

Reply:
I think not, because in our internal use case, it was originally receiving a boolean at `$absolute` argument and returning the same value untouched.
Additionally, the return value of this method  is intended to be used as argument for [`RoutingExtension::buildOptions()`](https://github.com/KnpLabs/KnpMenu/blob/b6aade272c345b6fbd07fce5929a761cba0909b8/src/Knp/Menu/Integration/Symfony/RoutingExtension.php#L21-L36) at "knplabs/knp-menu".
Even, if some project is using the method with other purpose and trusting the value of "routeAbsolute" being something different than a boolean, I guess we could dismiss such an edge case.